lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -709,65 +708,38 @@ def _get_hash_key(self, image: Image.Image, threshold: float) -> str:
 
 
 @timetrace("OCR")
-@requires_modules("pytesseract", extra="local-inference")
 def extract_ocr(
-    images: list[Image.Image], elements: list[list[Element]], ocr_images=False, ocr_tables=False, table_reader=None
+    images: list[Image.Image],
+    elements: list[list[Element]],
+    ocr_images: bool = False,
+    ocr_model_name: str = "easyocr",
 ) -> list[list[Element]]:
-    import pytesseract
-
+    if ocr_model_name not in EXTRACTOR_DICT.keys():
+        raise ValueError(f"Unknown ocr_model: {ocr_model_name}")
+    ocr_model: OcrModel = EXTRACTOR_DICT[ocr_model_name]()
     for i, image in enumerate(images):
-        width, height = image.size
-
         page_elements = elements[i]
-
+        width, height = image.size
         for elem in page_elements:
             if elem.bbox is None:
                 continue
             if elem.type == "Picture" and not ocr_images:
                 continue
-            # elif elem.type == "table" and not ocr_tables:
-            #     continue
-            elif elem.type == "table":
+            cropped_image = crop_to_bbox(image, elem.bbox)
+            if elem.type == "table":
+                tokens = []
```

Comment:
Maybe a function here?
```
elem.tokens = fixup_bboxes(elem, cropped_image)
```

Reply:
This isn't used anywhere else, we can do this when we resolve the TODO.

Reply:
I'll get to this in a follow-up PR, I'd like for the TODO to not be ignored.